h3/src/connection.rs

```diff
@@ -106,11 +153,33 @@ where
             .map_err(|e| Code::H3_STREAM_CREATION_ERROR.with_transport(e))?;
 
         let mut settings = Settings::default();
+
+        settings
+            .insert(
+                SettingId::MAX_HEADER_LIST_SIZE,
+                config.max_field_section_size,
+            )
+            .map_err(|e| Code::H3_INTERNAL_ERROR.with_cause(e))?;
+
+        settings
+            .insert(
+                SettingId::ENABLE_CONNECT_PROTOCOL,
+                config.enable_extended_connect as u64,
+            )
+            .map_err(|e| Code::H3_INTERNAL_ERROR.with_cause(e))?;
         settings
-            .insert(SettingId::MAX_HEADER_LIST_SIZE, max_field_section_size)
+            .insert(
+                SettingId::ENABLE_WEBTRANSPORT,
+                config.enable_webtransport as u64,
+            )
             .map_err(|e| Code::H3_INTERNAL_ERROR.with_cause(e))?;
+        settings
+            .insert(SettingId::H3_DATAGRAM, config.enable_datagram as u64)
+            .map_err(|e| Code::H3_INTERNAL_ERROR.with_cause(e))?;
+
+        tracing::debug!("Sending server settings: {settings:#x?}");
```

Comment:
CI says this is breaking MSRV. 
Is this log duplicate with the same in Line 228 of this file?

Reply:
It is the usage of inline formatting.

Will fix though running cargo check on my end with 1.57 gives a "failed to select version for `time`"

Reply:
Fixed :)